styles/documentation.css

```diff
@@ -0,0 +1,56 @@
+.component {
+    border-bottom: 1px solid var(--secondary-color);
+}
+
+.component:last-child {
+    border-bottom: none;
+}
+
+.component-heading {
+  font-size: 3rem;
+}
+
+.about-component,
+.example-description {
+  padding: var(--sm-spacing) 0 var(--default-spacing);
+  text-align: justify;
+}
+
+.example-heading {
+  font-size: 1.5rem;
+}
+
+.component-example {
+  border: 1px solid var(--primary-color-light-shade2);
+  border-radius: var(--default-border);
+  background-color: var(--basic-bg);
+  overflow: auto;
+  max-width: 100%;
+  /* Hide scrollbar for IE, Edge and Firefox */
+  -ms-overflow-style: none; /* IE and Edge */
+  scrollbar-width: none; /* Firefox */
+}
+
+.component-example::-webkit-scrollbar {
+  display: none;
+}
+
+.component-code > iframe {
+  width: 100% !important;
```

Comment:
Avoid using important.

Reply:
tried, wasn't able to avoid it

> Looks amazing ! a suggestion , in one of the code reviews with Vikram sir , he mentioned keeping contrast levels above certain level read here https://web.dev/color-and-contrast-accessibility/?utm_source=devtools The yellow on white might not be very readable and the contrast tool will help you check the correct color ![image](https://user-images.githubusercontent.com/75421090/155015878-73c13d6c-db93-452a-8405-233fc41429a1.png) Onkar Deshpande

thank you for pointing it out. I do plan to change the primary color in future 👍